src/action-ros-ci.ts

```diff
@@ -80,7 +80,7 @@ export async function execBashCommand(
 ): Promise<number> {
 	commandPrefix = commandPrefix || "";
 	const bashScript = `${commandPrefix}${commandLine}`;
-	const message = log_message || `Invoking "bash -c '${bashScript}'`;
+	const message = log_message || `Invoking  "bash -c '${bashScript}'`;
```

Comment:
extra spaces in this file intentional?

Reply:
ha. No - I was only adding whitespace to this file to test the rebuild hook.